ui/src/main/webapp/app/config/config-service.js

```diff
@@ -67,6 +67,7 @@ angular.module('app').factory('config', ['$http', '$q', 'defaultRestEndpoint', '
                         }
                         angular.merge(mergedConfig, customConfig);
                         config = mergedConfig;
+                        document.title = config.title || 'Gaffer';
```

Comment:
This default title should just be moved into the defaultConfig.json file

Reply:
Done.